nav2_recoveries/plugins/back_up.cpp

```diff
@@ -120,11 +120,12 @@ bool BackUp::isCollisionFree(
   double sim_position_change;
   const double diff_dist = abs(command_x_) - distance;
   const int max_cycle_count = static_cast<int>(cycle_frequency_ * simulate_ahead_time_);
-
+  geometry_msgs::msg::Pose2D init_pose = pose2d;
+  
```

Comment:
12:48:51 3: /tmp/ws/src/navigation2/nav2_recoveries/plugins/back_up.cpp:124:  Line ends in whitespace.  Consider deleting these extra spaces.  [whitespace/end_of_line] [4]

Linting issue as well